packages/remix-dev/compiler/plugins/serverBareModulesPlugin.ts

```diff
@@ -108,6 +110,7 @@ export function serverBareModulesPlugin(
           case "cloudflare-pages":
           case "cloudflare-workers":
           case "deno":
+          case "netlify-edge":
```

Comment:
Instead of adding this line to our compiler, use `serverDependenciesToBundle` in your template config to automatically bundle everything.